tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -415,7 +415,9 @@ Verify that the amended methodology is visible on the public methodologies page
     user scrolls down    400
     user checks page contains link with text and url
     ...    ${PUBLICATION_NAME} - Amended methodology
-    ...    ${PUBLIC_METHODOLOGY_URL_ENDING}
+    ...    ${PUBLIC_METHODOLOGY_URL_ENDING}-amended-methodology    # Slug has changed
+
+# @MarkFix visit old slug and get redirected to methodology
```

Comment:
Does this need addressing once Amy's frontend work is available?  Does this need a ticket ref?

Reply:
My plan is to do this after Amy's PR gets merged and I can rebase onto it. There will be a few other changes when I do that as well.

Reply:
This UI test has been written and commited now. So when rebasing on Amy's branch we'll just need to enable her work per the `middleware.ts` comment.